at_client/src/main/java/org/atsign/client/cli/Register.java

```diff
@@ -24,16 +24,16 @@ public static void main(String[] args) throws Exception {
         String otp;
         String validationResponse;
         String cramSecret;
-        String rootDomain = configReader.getProperty("ROOT_DOMAIN");
-        String rootPort = configReader.getProperty("ROOT_PORT");
-        String registrarUrl = configReader.getProperty("REGISTRAR_URL");
-        String apiKey = configReader.getProperty("API_KEY");
-        
-        if (rootDomain == null || rootPort == null || registrarUrl == null || apiKey == null){
+        String rootDomain = configReader.getProperty("rootDomain");
+        String rootPort = configReader.getProperty("rootPort");
+        String registrarUrl = configReader.getProperty("registrarUrl");
+        String apiKey = configReader.getProperty("apiKey");
```

Comment:
This is a breaking change. If the property is null when retrieving with the new name, then try to retrieve using the old name

Consider also adding some structure to the yaml; now's the best time to do this before it gets fossilized by usage

e.g.

root:
  domain: "xxxx"
  port: nnn
registrar:
  url: "https://..."
  apiKey: "xyz456abc123..."